src/test/java/com/cocodan/triplan/schedule/service/TestDataFactory.java

```diff
@@ -0,0 +1,27 @@
+package com.cocodan.triplan.schedule.service;
+
+import com.cocodan.triplan.schedule.dto.request.DailyScheduleSpotCreationRequest;
+import com.cocodan.triplan.schedule.dto.request.Position;
+import com.cocodan.triplan.schedule.dto.request.ScheduleCreationRequest;
+
+import java.time.LocalDate;
+import java.util.List;
+
+public class TestDataFactory {
+
+    public static ScheduleCreationRequest createScheduleCreation() {
```

Comment:
```suggestion
    public static ScheduleCreationRequest createScheduleCreationRequest() {
```
는 어떠신가요.

앞뒤로 create에서 끝나니까 한눈에 딱 들어오기 어려운 것 같아요.

Reply:
수정하겠습니다~